package.json

```diff
@@ -2,6 +2,11 @@
   "scripts": {
     "test": "xo"
   },
+  "dependencies": {
+    "grunt": "^1.0.1",
+    "grunt-cli": "^1.2.0",
+    "grunt-webstore-upload": "^0.8.10"
+  },
```

Comment:
Travis CI uses a "production" environment by default. If we make these `devDependencies` we have to  `before_install: npm install --dev`.